### PR TITLE
(Review complete) Edits for 70007 (Examiner)

### PR DESCRIPTION
**Pull request by HotHeadEmperor**
Line 8 feels odd. Will look into it.

### CODE REVIEW

File: CSV/Language/English/Card/70007.txt [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
+ギルドに所属する中堅マスター。ハンタータイプのカードをメインに使用し、戦士やフクロウのカードをサポートに用いる。\n常日頃から谷間を露出させた格好で出歩いており、人に見られるのがわりと好きな性格。男の子が彼女の胸に見とれていたりすると、両手で寄せ上げてサービスポーズを見せつけたりしてくるので、性癖がゆがむ男の子が後を絶たない。\nランクはC級だが、試験のために実力を抑えている。,A mid-level cardmaster who works at the guild. She mainly uses soldier- and owl-type cards that are built around supporting her hunter-type cards. She enjoys being watched by others; always walking around in outfits that expose her ample cleavage. When boys stare at her tits; she teasingly pushes them together with her hands into a service pose; which drive many young boys mad. Despite being ranked C-class; she holds back her true strength for exams.
```
[the remark this answers is not in the document]
Comment (HotHeadEmperor) [reply CSV/Language/English/Card/70007.txt:4]: So in general, to my understanding, is if you have a list of things that should be hyphenated if they were singular, they are hyphenated in all instances.

Reference:
https://english.stackexchange.com/questions/113422/how-to-use-hyphens-appropriately-when-listing-multiple-hyphenated-terms

File: CSV/Language/English/Card/70007.txt [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
+おお、本当ですか！,Ooh; is that so?
 すばらしいです！まだ大会も始まったばかりだというのに、良いすべり出しですね！,That's wonderful to hear! The tournament has only just begun; and yet you're already off to a good start!
-それではギルドリングをこちらにお渡しください。ポイントを確認させていただきます。,Now then. Please hand me your guild ring. I shall confirm the number points you've collected.
+それではギルドリングをこちらにお渡しください。ポイントを確認させていただきます。,Now then, please hand me your guild ring. I shall confirm the number of points you've collected.
 ありがとうございます。確かに指輪には必要量のランクポイントが貯まっていますね。,Thank you. You certainly have the required rank points stored in your ring.
-これであなたはFランクからEランクに昇格できるわけですが…。,With this; you can be promoted from F rank to E rank...
+これであなたはFランクからEランクに昇格できるわけですが…。,With this; you can be promoted from F-rank to E-rank...
 ランク昇格の際には、１つ試験をこなしてもらわないといけません。,However; I must add in order to qualify for  a rank promotion; you must also pass a test.
-試験の内容はランクごとに異なります。Eランクの昇格試験は、ギルド職員との戦闘試験です。,The content of the test varies by rank. The E-rank promotion test is a battle test with guild staff.
+試験の内容はランクごとに異なります。Eランクの昇格試験は、ギルド職員との戦闘試験です。,The content of the test varies by rank. The E-rank promotion test is a test battle with guild staff.
```
[the remark this answers is not in the document]
Comment (HotHeadEmperor) [reply CSV/Language/English/Card/70007.txt:17]: Good call!

File: CSV/Language/English/Card/70007.txt [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
-ふふ、さすがに試験をすぐクリアしただけはありますね。\nとても頼もしいです。,Fufu. Well; you did clear the test immediately as I expected; I'm glad to know that we can rely on you.
+ふふ、さすがに試験をすぐクリアしただけはありますね。\nとても頼もしいです。,Fufu. Well; you did clear the test immediately as I expected. I'm glad to know that we can rely on you.
 それでは引き続き大会でのバトル頑張ってください！,Well then; please keep up the good work in the tournement!
 僕は色の変わったギルドリングをウキウキした心地で眺めながら、宿への帰路についた。,As I gazed eagerly at the newly colored guild ring; I made my way back to the inn.
 ほとんどカードも使えなかった昔を思えば、とても順調だ。これも師匠がしっかり鍛えてくれたからだろう。,Considering in the past there were times when I could barely use any cards; things are going quite smoothly now. It's probably because my mentor trained me well.
 試験官とのバトルを終えたからか、体はすっかり疲れていた。僕はこの先のことをうつらうつらと思いながら、眠りについたのだった。,Probably due to the battle with the examiner; my body was completely exhausted. I fell asleep; drowsily thinking about what lay ahead.
 …………。,...
-…。この景色。いつも見る夢か…。,...This scene. It's the same dreamI have every every time...
+…。この景色。いつも見る夢か…。,...This scene. It's the same dream I have every time...
```
[the remark this answers is not in the document]
Comment (HotHeadEmperor) [reply CSV/Language/English/Card/70007.txt:43]: I'll try it, you can see how it feels.

File: CSV/Language/English/Card/70007.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Giratena) [reply CSV/Language/English/Card/70007.txt:4]: Wow! Colour me surprised! I didn't know about this. Proceed as is then ^^

File: CSV/Language/English/Card/70007.txt [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
 はい,Yes
 いいえ,No
 いくつかのバトルを経て、ランクポイントが十分にたまった僕はギルドの建物を訪れていた。,After several battles; I had accumulated enough rank points and visited the guild building.
 あら、また会いましたね、新人マスターさん。ギルドへようこそ。大会での戦いは順調ですか？,Ah; it's the new cardmaster. We meet again! Welcome to the guild. How are the battles in the tournament going?
 ランクポイントがたまりました！,I've accumulated some rank points!
-おお、本当ですか！,Ooh; is that so!
+おお、本当ですか！,Ooh; is that so?
 すばらしいです！まだ大会も始まったばかりだというのに、良いすべり出しですね！,That's wonderful to hear! The tournament has only just begun; and yet you're already off to a good start!
-それではギルドリングをこちらにお渡しください。ポイントを確認させていただきます。,Now then. Please hand me your guild ring. I shall confirm the number points you've collected.
+それではギルドリングをこちらにお渡しください。ポイントを確認させていただきます。,Now then, please hand me your guild ring. I shall confirm the number of points you've collected.
```
[the remark this answers is not in the document]
Comment (Giratena) [reply CSV/Language/English/Card/70007.txt]: No worries at all ^^

**Pull request merged**